emvue_collect.py

```diff
@@ -0,0 +1,74 @@
+# SPDX-License-Identifier: BSD-3-Clause
+import datetime
+import json
+import pyemvue
+import sys
+import time
+
+from pyemvue.enums import Scale, Unit
+
+try:
+    with open('.user.json') as f:
+        user_auth = json.load(f)
+except FileNotFoundError:
+    sys.exit('Expected .user.json file with {"user": "...", "password": "..."}')
+
+vue = pyemvue.PyEmVue()
+vue.login(username=user_auth['user'], password=user_auth['password'],
```

Comment:
There is another login method that uses the tokens generated from a standard login. Is it better to use the token login when it works? Not sure if this is better for the API servers or not.

Reply:
The token based login appears to be for re-authentication not initial login... not sure how long it sticks arround.